Makefile

```diff
@@ -116,23 +116,19 @@ build: ## By default we should build in release mode
 
 .PHONY: build-no-std
 build-no-std: ## Build without the standard library
-	$(BUILD_GENERATED_FILES_IN_SRC) cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib --exclude bench-prover
+	$(BUILD_GENERATED_FILES_IN_SRC) cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib
 
 
 .PHONY: build-no-std-testing
 build-no-std-testing: ## Build without the standard library. Includes the `testing` feature
-	$(BUILD_GENERATED_FILES_IN_SRC) cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude miden-bench-tx --features testing --exclude bench-prover
+	$(BUILD_GENERATED_FILES_IN_SRC) cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude bench-transaction --features testing
 
 # --- benchmarking --------------------------------------------------------------------------------
 
 .PHONY: bench-tx
 bench-tx: ## Run transaction benchmarks
-	cargo run --bin bench-tx
-
-.PHONY: bench-prover
-bench-prover: ## Run prover benchmarks and consolidate results.
-	cargo bench --bin bench-prover --bench benches
-	cargo run --bin bench-prover
+	cargo run --bin bench-transaction --features concurrent
+	cargo bench --bin bench-transaction --bench time_counting_benchmarks --features concurrent
```

Comment:
@PhilippGackstatter Strange, both my local and server versions has this change. Could it be an issue with an old cache or something? I made this change in `refactor: update README, update folder names` commit, so it should be pushed. Let me know if you still have an outdated version, I will try to push it again.